mne/viz/_3d.py

```diff
@@ -444,10 +480,10 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
         mesh.data.cell_data.normals = None
         mlab.pipeline.surface(mesh, color=colors[key], opacity=alphas[key])
 
-    datas = (eeg_loc, hpi_loc, car_loc, ext_loc)
-    colors = ((1., 0., 0.), (0., 1., 0.), (1., 1., 0.), (1., 0.5, 0.))
-    alphas = (1.0, 0.5, 0.5, 0.25)
-    scales = (0.005, 0.015, 0.015, 0.0075)
+    datas = (eeg_loc, eegp_loc, hpi_loc, car_loc, ext_loc)
+    colors = ((1., 0, 0), (0.5, 0, 0), (0., 1, 0), (1., 1, 0), (1, 0.5, 0))
```

Comment:
I don't like that the "projected" is so dissimilar from the "original" red, but most of the similar colors are actually taken by other things. So I tweaked `(0, 0.9, 1)` to `(0, 0.5, 1)` and it looks more MNE-C like (and still good I think; ignore the blockiness/normal problems, that's my own mayavi build issue):

![screenshot from 2016-10-28 19-13-00](https://cloud.githubusercontent.com/assets/2365790/19825021/9059d3ac-9d42-11e6-895b-8bc8d0a2da8d.png)


Okay?